includes/admin/class-wc-rest-stripe-settings-controller.php

```diff
@@ -196,6 +196,10 @@ public function validate_statement_descriptor( $value, $request, $param, $max_le
 			return $string_validation_result;
 		}
 
+		if ( '' === $value ) {
```

Comment:
```suggestion
		// Relaxing validation because it's blocking the user from saving when they're on another tab
		// TODO: work that out with either with a UX approach or handling the validations of each tab separately
		if ( '' === $value ) {
```